recipe/meta.yaml

```
@@ -20,34 +20,35 @@ outputs:
 
     requirements:
       host:
-        - python >=3.8
+        - python >=3.9
         - pip
+        - rdkit
       run:
-        - python >=3.8
+        - python >=3.9
         - openff-toolkit-base >=0.11.0
         - numpy
-        - pydantic <2.0.0a0
+        - pydantic <3.0.0a0
         - networkx
         - jinja2
```

```suggestion
        - jinja2
        - rdkit
```

Perhaps try it in `run` as well? It doesn't look like rdkit is installed in the test environment.